index.html

```diff
@@ -0,0 +1,8 @@
+<html>
+  <head>
+  </head>
```

Comment:
There should be a title here

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Reusable

Reply:
I didn't think we need a title. It's just a simple hello world.